extensions/interactions/interaction_specs.json

```diff
@@ -747,6 +747,13 @@
           ]
         },
         "default_value": 1
+    }, {
+      "description": "Catch Misspellings (Detect if answer is misspelled by atmost 2 letters)",
```

Comment:
I think this doesn't quite explain to the exploration creator what happens if the answer satisfies this criterion. Do you tell them this anywhere (so that they can understand what is going on)?

Reply:
Fixed.